arcade/__init__.py

```diff
@@ -37,8 +37,8 @@ def configure_logging(level: Optional[int] = None):
 
 def get_timestamp(
         how: str = "%Y_%m_%d_%H%M_%S_%f%Z",
-        when: Optional[types.HasStrftime] = None,
-        tzinfo: Optional[datetime.tzinfo] = None
+        when: Optional[types.HasStrftime | datetime] = None,
+        tzinfo: Optional[_TzInfo] = None
```

Comment:
I think I removed a top-level `datetime` import but left behind the broken property `datetime.tzinfo` line earlier. With the import suggestion at the top, I think this should work.
```suggestion
        tzinfo: Optional[tzinfo] = None
```